editor/edit-post/modes/visual-editor/inserter.js

```diff
@@ -40,6 +41,10 @@ export class VisualEditorInserter extends Component {
 		onInsertBlock( createBlock( name ) );
 	}
 
+	isDisabledBlock( block ) {
+		return block.useOnce && find( this.props.blocks, ( { name } ) => block.name === name );
```

Comment:
Minor: `some` ([Array#some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)) would be more appropriate than `find` ([Array#find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)) only because we care that _it exists_ (boolean), and have no intent to use the found value.

Reply:
Can I still commit in this branch to do that?

Reply:
>Can I still commit in this branch to do that?

I'm not sure how forked branches work when a merge has already been done upstream†, but at the very least you'd still need to create a new pull request if you feel compelled to make the change.

† I'm kinda curious though if committing to your branch would allow for a new pull request 😄

Reply:
I did a commit but nothing got happen here. I think I need to create a new pull request... xD